plugins/plugin-chart-echarts/src/Timeseries/Regular/controlPanel.tsx

```diff
@@ -89,6 +89,7 @@ const config: ControlPanelConfig = {
     sections.advancedAnalyticsControls,
     sections.annotationsAndLayersControls,
     sections.forecastIntervalControls,
+    { ...sections.titleControls, expanded: true },
```

Comment:
avoid clone titleControls when defined `expanded: true` in section.
```
sections.titleControls,
```

Reply:
Done